README.md

```diff
@@ -4,6 +4,42 @@
 
 > This is my first project in the Microverse program.
 
+![screenshot](./Screenshot_3.png)
+
+In this project, we covered some basic things in github
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+No Live Preview
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/vladanvi99)
+- Twitter: [@twitterhandle](https://twitter.com/vladanvi99)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/vladan-videnovi%C4%87-780bb11b2/)
```

Comment:
[OPTIONAL] 

- [ ] the `githubhandle` is just a placeholder within the template, feel free to change it with your own username.
[@vladanvi99](https://github.com/vladanvi99)